packages/admin/admin/src/mui/menu/Item.styles.ts

```diff
@@ -59,44 +75,82 @@ export const styles = (theme: Theme) =>
-                color: "#fff",
+                backgroundColor: theme.palette.common.white,
+                color: theme.palette.primary.main,
+                fontWeight: theme.typography.fontWeightBold,
                 "&:after": {
-                    backgroundColor: theme.palette.primary.dark,
+                    backgroundColor: ({ isMenuOpen }) => (isMenuOpen ? theme.palette.primary.main : undefined),
                 },
                 "&:hover": {
-                    backgroundColor: theme.palette.primary.dark,
+                    backgroundColor: theme.palette.grey[50],
                 },
                 "& [class*='MuiListItemText-primary']": {
                     fontWeight: theme.typography.fontWeightBold,
                 },
+                "& [class*='MuiListItemIcon-root']": {
+                    color: theme.palette.primary.main,
+                },
             },
             "& [class*='MuiListItemText-root']": {
                 margin: 0,
             },
-            "& [class*='MuiListItemText-primary']": {
-                fontWeight: theme.typography.fontWeightRegular,
-                fontSize: 14,
-                lineHeight: "20px",
-            },
         },
         level3: {
             color: colors.textLevel2,
-            paddingLeft: 20,
+            paddingLeft: ({ isMenuOpen }) => (isMenuOpen ? 50 : 20),
             paddingRight: 20,
             paddingTop: 0,
             paddingBottom: 0,
             position: "relative",
+            "& [class*='MuiListItemText-inset']": {
+                paddingLeft: 0,
+            },
+            "&:last-child": {
+                borderBottom: ({ isMenuOpen, level }) => (isMenuOpen && level !== 3 ? `1px solid ${theme.palette.grey[50]}` : "initial"),
+                boxSizing: "border-box",
+            },
+            "&[class*='Mui-selected']": {
+                backgroundColor: theme.palette.common.white,
+                color: theme.palette.primary.main,
+                fontWeight: theme.typography.fontWeightBold,
+                "&:hover": {
+                    backgroundColor: theme.palette.grey[50],
+                },
+                "& [class*='MuiListItemText-primary']": {
+                    fontWeight: theme.typography.fontWeightBold,
+                },
+                "& [class*='MuiListItemIcon-root']": {
+                    color: theme.palette.primary.main,
+                },
+            },
+            "& [class*='MuiListItemText-root']": {
+                margin: 0,
+            },
+            "& [class*='MuiListItemText-primary']": {
+                fontWeight: theme.typography.fontWeightRegular,
+                fontSize: 14,
+                lineHeight: "20px",
+                paddingLeft: ({ isMenuOpen }) => (isMenuOpen ? 14 : 0),
+                paddingTop: 14,
+                paddingBottom: 14,
+            },
+        },
+        level3Enumeration: {
```

Comment:
What's the difference to `level3`?

Reply:
It takes care of these lines before the element:

![image](https://github.com/vivid-planet/comet/assets/56400587/361d21a8-5c9a-4114-a4b9-f4315af9682e)